README.md

```diff
@@ -72,124 +72,17 @@ use it with other tools: http://localhost:8001/openapi.json
 As a proof-of-concept, an object model called `Node` is defined in this API.
 It's possible to create new objects and retrieve them via the API.
 
-### Creating a Node
```

Comment:
We can probably just drop the Node and Pub/Sub parts from the README as it's already pretty specialised. If we mention this here we would need to mention all the aspects of the API as a table of contents, which is something we'll have in the main docs anyway.

Reply:
Agreed. Updated the README file.